### DISCUSSION TITLE
Fief Components for Angular?

### DISCUSSION THREAD

**Discussion by ScottFred** [fief-dev/fief#310] [General]
Since there appears to be an Angular renaissance happening, are there future plans or desire to support Angular similar to the support shown for React, e.g. documentation, component installation, schematics?

**Comment by frankie567** [comment]
I have to admit it's been a while since I've worked with Angular. I'm not reluctant to the idea but I can't promise I'll be able to support that *soon*. I also have lot of [demands for VueJS support](https://github.com/orgs/fief-dev/discussions/238) as well 😄
